src/client/user/usersReducer.js

```diff
@@ -310,7 +308,11 @@ export default function usersReducer(state = initialState, action) {
   }
 }
 
-export const getUser = (state, username) => get(state.users, getUserDetailsKey(username), {});
+export const getAllUsers = state => get(state, 'users', {});
+export const getUser = (state, username) => {
+  const allUsers = getAllUsers(state);
+  return get(allUsers, username, {});
+};
```

Comment:
you can use only _.get():
export const getUser = (state, username) => get(state, ['users', username], {});